include/mbedtls/config.h

```diff
@@ -3535,6 +3535,22 @@
  */
 //#define MBEDTLS_PLATFORM_GMTIME_R_ALT
 
+/*
+ * \def MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED
+ *
+ * Enable the verified implementations of crypto primitives
+ * from Project Everest (currently only Curve25519).
+ * This feature breaks ECDH backward compatibility (see also
+ * MBEDTLS_ECDH_LEGACY_CONTEXT).
+ *
+ */
+#define MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED
+
+/* \} name SECTION: Customisation configuration options */
+
+/* Target and application specific configurations */
+//#define YOTTA_CFG_MBEDTLS_TARGET_CONFIG_FILE "target_config.h"
```

Comment:
In commit "ECDH: Add Everest Curve25519 config.h option", don't add yotta things.

Reply:
This is removed later, I thought about fixing up this commit, but decided to be faithful to the original PR.

Reply:
OK...

Reply:
(I fixed it up because now I don't see any value in having them separately either, I was just explaining my original thinking.)